holoviews/plotting/bokeh/element.py

```diff
@@ -538,17 +538,44 @@ def __init__(self, element, plot=None, **params):
 
     def initialize_plot(self, ranges=None, plot=None, plots=None):
         self.mplplot.initialize_plot(ranges)
-        plot = mpl.to_bokeh(self.mplplot.state)
+
```

Comment:
This is a general purpose wrapper class so there's only this one and the one that converts matplotlib plots to RGB images (which we should probably remove). Currently we only register the seaborn Element types for actual conversion and have the matplotlib 3D plots registered to use the RGB conversion. I'll actually go ahead and delete these:

```                Surface: PlotSelector(lambda x: 'bokeh',
                                      [('mpl', SurfacePlot),
                                       ('bokeh', BokehMPLRawWrapper)], True),
                Scatter3D: PlotSelector(lambda x: 'bokeh',
                                        [('mpl', Scatter3DPlot),
                                         ('bokeh', BokehMPLRawWrapper)], True),
                Trisurface: PlotSelector(lambda x: 'bokeh',
                                         [('mpl', TrisurfacePlot),
                                          ('bokeh', BokehMPLRawWrapper)], True)},
```

 as part of this PR.